PokemonGo.RocketAPI/Login/GoogleLogin.cs

```diff
@@ -1,86 +1,49 @@
-﻿using System.Collections.Generic;
-using System.Threading;
-using System.Threading.Tasks;
-using PokemonGo.RocketAPI.Helpers;
-
-namespace PokemonGo.RocketAPI.Login
-{
-    public static class GoogleLogin
-    {
-        private const string OauthTokenEndpoint = "https://www.googleapis.com/oauth2/v4/token";
-        private const string OauthEndpoint = "https://accounts.google.com/o/oauth2/device/code";
-        private const string ClientId = "848232511240-7so421jotr2609rmqakceuu1luuq0ptb.apps.googleusercontent.com";// "848232511240-73ri3t7plvk96pj4f85uj8otdat2alem.apps.googleusercontent.com";
-
-        private const string ClientSecret = "NCjF1TLi2CcY6t5mt0ZveuL7";
-
-
-        public static async Task<TokenResponseModel> GetAccessToken(DeviceCodeModel deviceCode)
-        {
-            int count = 0;
-
-            //Poll until user submitted code..
-            TokenResponseModel tokenResponse;
-            do
-            {
-                await Task.Delay(2000);
-                tokenResponse = await PollSubmittedToken(deviceCode.device_code);
-                count++;
-            } while (tokenResponse.access_token == null || tokenResponse.refresh_token == null && count < 100);
-
-            return tokenResponse;
-        }
-
-        public static async Task<DeviceCodeModel> GetDeviceCode()
-        {
-            var deviceCode = await HttpClientHelper.PostFormEncodedAsync<DeviceCodeModel>(OauthEndpoint,
-                new KeyValuePair<string, string>("client_id", ClientId),
-                new KeyValuePair<string, string>("scope", "openid email https://www.googleapis.com/auth/userinfo.email"));
-           return deviceCode;
-        }
-
-        private static async Task<TokenResponseModel> PollSubmittedToken(string deviceCode)
-        {
-            return await HttpClientHelper.PostFormEncodedAsync<TokenResponseModel>(OauthTokenEndpoint,
-                new KeyValuePair<string, string>("client_id", ClientId),
-                new KeyValuePair<string, string>("client_secret", ClientSecret),
-                new KeyValuePair<string, string>("code", deviceCode),
-                new KeyValuePair<string, string>("grant_type", "http://oauth.net/grant_type/device/1.0"),
```

Comment:
No, I removed everything in "GoogleLogin.cs", because it is no longer used and moved the login code from "GoogleLoginGPSOAuth.cs" to GoogleLogin, so it is clear that there are only 2 types of login in use: PTC and Google.